.github/workflows/pytest-core-nompi.yml

```diff
@@ -105,6 +106,13 @@ jobs:
           language: "openmp"
           sympy: "1.13"
 
+        - name: pytest-ubuntu-py313-gcc14-omp
```

Comment:
@FabioLuporini this doesn't touch anything it just makes python 3.13 accepted, it's the default now on osx